frontend/src/components/sequenceEditOptionsTab/ImageOptionsTab.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div class="flex flex-row items-center space-x-5 w-full pt-5">
+    <ButtonComponent @click="selectImage()">Bild ändern</ButtonComponent>
+    <div class="space-x-2 flex flex-row items-center w-96">
+      <p>Skalierung:</p>
+      <input
+        type="range"
+        min="0"
+        max="1"
+        step="0.01"
+        v-model="refContent.scale"
+        class="h-2 rounded-full appearance-none cursor-pointer bg-interactable-border grow"
+        @change="updateScale()"
+      />
+      <p>{{ (refContent.scale * 100).toFixed(0) }}%</p>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { PropType, ref, watch } from 'vue';
+import { ImageContent } from '../../../../model/slide/content/ImageContent';
+import ButtonComponent from '../ButtonComponent.vue';
+
+const props = defineProps({
+  /**
+   * The image content to edit
+   */
+  imageContent: {
+    type: Object as PropType<ImageContent>,
+    required: true,
+  },
+});
+
+const emits = defineEmits([
+  /**
+   * Emitted when the content has been edited
+   *
+   * @param content the updated content
+   */
+  'update-content',
+]);
+
+const refContent = ref(props.imageContent);
+
+/**
+ * Opens a file dialog to select a new image.
+ */
+function selectImage(): void {
+  let input = document.createElement('input');
```

Comment:
Can we limit the filesize for this? We don't want teachers to upload 40gb 4k images

Reply:
should be easily possible. what size to we want as max?

Reply:
I'd suggest something like 2MB maybe? Should be plenty I think